code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -127,19 +128,21 @@ GLOBAL_LIST_EMPTY(damage_icon_parts)
 
 	previous_damage_appearance = damage_appearance
 
-	var/icon/standing = new /icon('icons/mob/dam_human.dmi', "00")
+	var/icon/standing = new('icons/mob/dam_human.dmi', "00")
 
-	var/image/standing_image = new /image("icon" = standing, "layer" =-DAMAGE_LAYER)
+	var/image/standing_image = image("icon" = standing, "layer" = DAMAGE_LAYER)
```

Comment:
Should the `-` have been removed here? Every other standing cache uses the negative values.